models/ed/R/met2model.ED2.R

```diff
@@ -196,27 +191,14 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
       print("Skipping to next year")
       next
     }
-    
-    
+
+
     ## calculate potential radiation in order to estimate diffuse/direct
-    f                <- pi/180 * (279.5 + 0.9856 * doy)
-    et               <- (-104.7 * sin(f) + 596.2 * sin(2 * f) + 4.3 * sin(4 * f) - 429.3 * 
-                           cos(f) - 2 * cos(2 * f) + 19.3 * cos(3 * f)) / 3600  # equation of time -> eccentricity and obliquity
-    merid            <- floor(lon/15) * 15
-    merid[merid < 0] <- merid[merid < 0] + 15
-    lc               <- (lon - merid) * -4 / 60  ## longitude correction
-    tz               <- merid / 360 * 24  ## time zone
-    midbin           <- 0.5 * dt / 86400 * 24  ## shift calc to middle of bin
-    t0               <- 12 + lc - et - tz - midbin  ## solar time
-    h                <- pi/12 * (hr - t0)  ## solar hour
-    dec              <- -23.45 * pi/180 * cos(2 * pi * (doy + 10) / 365)  ## declination
-    
-    cosz             <- sin(lat * pi/180) * sin(dec) + cos(lat * pi/180) * cos(dec) * cos(h)
-    cosz[cosz < 0]   <- 0
-    
+    cosz <- PEcAn.data.atmosphere::solar_angle(doy, lat, lon, dt)
```

Comment:
does this function return solar_angle, or cos(zenith angle)? I would expect that solar_angle to return zenith_angle, and perhaps function should be named cos_solar_zenith_angle???